packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -709,6 +697,16 @@ export default {
             window.location.assign(`${this.paymentPageUrlPrefix}/${this.orderId}`);
         },
 
+        /**
+         * Call update checkout with only the user's DOB for age verification
+         * This is to avoid creating too many side effects with the original mapper for update checkout
+         */
+        async verifyCustomerAge () {
+            const data = this.mapCheckoutUpdateRequest(true);
```

Comment:
The argument `true` here looks a bit odd... have you considered setting a default param on the method?

Reply:
I agree. I've changed it to an `options` object parameter